abstraction/src/main/java/net/wesjd/anvilgui/version/VersionWrapper.java

```diff
@@ -91,7 +83,16 @@ public interface VersionWrapper {
      * @param title  The title of the anvil inventory
      * @return The Container instance
      */
-    Object newContainerAnvil(Player player, Object title);
+    AnvilContainerWrapper newContainerAnvil(Player player, Object title);
+
+    /**
+     * Checks if the current Minecraft version actually supports custom titles
+     *
+     * @return The current supported state
+     */
+    default boolean isCustomTitleSupported() {
```

Comment:
Should this default to true? Should we not make the implementation explicitly set this? It feels error prone that an implementation could accidentally say that custom titles are supported.

Reply:
I made this method return true by default because every new version to come supports titles, but a never changing set of versions (1.7 - 1.13) does not, an exception.